robottelo/ssh.py

```diff
@@ -237,7 +243,7 @@ def download_file(remote_file, local_file=None, hostname=None):
 
 
 def command(cmd, hostname=None, output_format=None, username=None,
-            password=None, key_filename=None, timeout=10):
+            password=None, key_filename=None, timeout=120, conn_timeout=10):
```

Comment:
this is a preventive functionality from hanging commands, so we can put as much as we can (with a reasonable value of course), so will put 300 everywhere to be compatible with API